libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKAnalyticsPublisher.h

```diff
@@ -31,12 +31,14 @@
 
 @protocol SFSDKAnalyticsPublisher <NSObject>
 
+typedef void (^ _Nonnull PublishCompleteBlock)(BOOL success);
```

Comment:
Should we add the Error too, for the case where there is an error?  The consumer could make use of it.